types/npmcli__promise-spawn/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
IMO this one is not required really, @types/node are installed by automatic types acquisition (in supporting IDEs) or explicitly by user.

Reply:
Fixed.